elements/pfe-accordion/src/pfe-accordion-header.scss

```diff
@@ -63,3 +64,39 @@ button[aria-expanded="true"] {
     @include pfe-accordion-variables($state: expanded, $theme: #{$context});
   }
 }
+
+/// ===========================================================================
+/// DISCLOSURE STYLES
+/// ===========================================================================
+
+:host([pfe-disclosure="true"]) {
+  button {
+    @include pfe-accordion-variables($variant: disclosure);
+    padding: #{pfe-var(container-spacer)} calc(#{pfe-var(container-spacer)} * 1.5) #{pfe-var(container-spacer)} 50px !important;
```

Comment:
Don't forget to update variable and not reset properties. 👍

Reply:
added a `triggerPadding` variable.